src/app/devices/deviceList/components/addDevice/components/addDevice.tsx

```diff
@@ -72,7 +71,7 @@ export default class AddDevice extends React.Component<AddDeviceActionProps & Ad
                 {(context: LocalizationContextInterface) => (
                      <form onSubmit={this.handleSave} className="view add-device">
                         <div className="view-header">
-                            <Route component={BreadcrumbContainer} />
+                            <Breadcrumb/>
```

Comment:
is not it a nested route inside deviceList? why do you need to mount breadcrumb here explicitly?